doc/api/index.rst

```diff
@@ -218,6 +219,7 @@ and store them in the GMT cache folder.
     datasets.load_mars_shape
     datasets.load_ocean_ridge_points
     datasets.load_sample_bathymetry
+    datasets.load_sample_dataframe
```

Comment:
If we're deprecating the other load_* functions, do we need to keep them here? Or add a 'Deprecated' note somewhere?

Reply:
I prefer to add "deprecated" notes in the docstrings of these functions.